install/install.sh

```diff
@@ -50,6 +50,11 @@ docker --version || curl -fsSL https://get.docker.com | env -u VERSION sh || (
 )
 systemctl enable docker
 
+#creating a Unix group called docker and adding users to it.
+echo "Created the docker group"
```

Comment:
"Creating the docker group." seems to replicate better the other messages writing style.

Also, this command is in fact adding the default user to the pre-existing docker group, so something like "Adding default user to 'docker' group." seems better.